templates/hobbygroups/index.html

```diff
@@ -0,0 +1,214 @@
+					<h3>Makrellgutta</h3>
+					<p>
+                        Makrellgutta er interessegruppen for alle som liker markell, tomat eller deres edlere kombinasjon!
+                        <ul>
+							<li><a href="/wiki/online/info/innsikt-og-interface/interessegrupper/makrellgutta/">Les mer om oss her!</a></li>
+					    </ul>
+                    </p>
+				</div>
+			</div>
+			<div class="col-xs-12 col-sm-6 col-md-6">
+				<div class="col-md-5">
+					<img src="{{ STATIC_URL }}img/hobbygroups/sport.png" alt="sport" />
+				</div>
+				<div class="col-md-7 hobbydescription">
+					<h3>X-Sport</h3>
+					<p>
+						Hovedoppgaven til interessegruppen er å få samlet mennesker for å spille squash, dra å kite, låne hall for å spille håndball osv.
+                        Dvs. sporter/idretter der man gjerne trenger å være flere for å kunne drive med. Vi skal etterstrebe å ha et arrangement i uken.
+                        <ul>
+							<li><a href="/wiki/online/info/innsikt-og-interface/interessegrupper/x-sport/">Les mer om oss her!</a></li>
+                            <li><a href="https://www.facebook.com/groups/xsportonline/">Du finner oss også på facebook</a></li>
+					    </ul>
+                    </p>
+				</div>
+			</div>
+		</div>
+
+        <div class="row">
+			<div class="col-xs-12 col-sm-6 col-md-6">
+				<div class="col-md-5">
+					<img src="{{ STATIC_URL }}img/hobbygroups/c_sharp.jpeg" alt="c#" />
+				</div>
+				<div class="col-md-7 hobbydescription">
+					<h3>Linjekoret C#</h3>
+					<p>
+						Linjekoret C# er Online interessegruppen for de som liker å synge, som også er eneste kravet for å være med.
+                        Koret er en delt interessegruppe med linjeforeningen Abakus så hvis du har lyst til å
+                        bli kjent med personer fra Abakus er dette en gylden mulighet.
+                        <ul>
+                            <li><a href="/wiki/online/info/innsikt-og-interface/interessegrupper/koret/">Les mer om oss her!</a></li>
+					    </ul>
+                    </p>
+				</div>
+			</div>
+			<div class="col-xs-12 col-sm-6 col-md-6">
+				<div class="col-md-5">
+					<img src="{{ STATIC_URL }}img/hobbygroups/ontap.png" alt="ontap" />
+				</div>
+				<div class="col-md-7 hobbydescription">
+					<h3>Ontap</h3>
+					<p>
+                        Vi er en gjeng bryggeinteresserte informatikkstudenter på NTNU!
+                        Har du lyst til å være med å brygge Onlines beste øl?
+                        <ul>
+							<li><a href="/wiki/online/info/innsikt-og-interface/interessegrupper/ontap/">Les mer om oss her!</a></li>
+					    </ul>
+                    </p>
+				</div>
+			</div>
+		</div>
+
+        <div class="row">
+			<div class="col-xs-12 col-sm-6 col-md-6">
+				<div class="col-md-5">
+					<img src="{{ STATIC_URL }}img/hobbygroups/strikk.png" alt="Strikk og Drikk" />
+				</div>
+				<div class="col-md-7 hobbydescription">
+					<h3>Strikk & drikk</h3>
```

Comment:
Use `&amp;` instead, because some browsers might try to parse `&` and unexpected results might occur.